bigquery_etl/query_scheduling/task.py

```diff
@@ -284,6 +285,19 @@ def of_multipart_query(cls, query_file, metadata=None, dag_collection=None):
         task.sql_file_path = os.path.dirname(query_file)
         return task
 
+    @classmethod
+    def of_script(cls, query_file, metadata=None, dag_collection=None):
+        """
+        Create task that schedules the corresponding script in Airflow.
+
+        Raises FileNotFoundError if not metadata file exists for query.
+        If `metadata` is set, then it is used instead of the metadata.yaml
+        file that might exist alongside the query file.
+        """
+        task = cls.of_query(query_file, metadata, dag_collection)
+        task.sql_file_path = os.path.dirname(query_file)
```

Comment:
Shouldn't this point to the `script.sql` file instead of the directory since `run_query` expects a query file?